package.json

```diff
@@ -37,7 +37,7 @@
     "@mongodb-js/zstd": "^1.1.0",
     "gcp-metadata": "^5.2.0",
     "kerberos": "^2.0.1",
-    "mongodb-client-encryption": ">=2.3.0 <3",
+    "mongodb-client-encryption": ">=6.0.0-alpha.0 <7",
```

Comment:
Optional, but probably worth doing.  After my previous PR, the old peer dependency compatibility range wasn't accurate.